Source/UI/CommonUI/View/Component/InputView.swift

```diff
@@ -168,12 +174,14 @@ extension InputView {
 
 extension InputView: TextFieldViewDelegate {
     func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
-        delegate?.textField(textField, shouldChangeCharactersIn: range, replacementString: string) ?? true
+        let shouldChangeCharacter = delegate?.textField(textField, shouldChangeCharactersIn: range, replacementString: string) ?? true
+        textFieldContainerBorder.layer.borderColor = style?.textfield.focusBorderColor.cgColor
+        return shouldChangeCharacter
```

Comment:
```suggestion
        textFieldContainerBorder.layer.borderColor = style?.textfield.focusBorderColor.cgColor
        return delegate?.textField(textField, shouldChangeCharactersIn: range, replacementString: string) ?? true
```

Reply:
I have tried this suggest and color still gray, 
`textFieldContainerBorder.layer.borderColor = style?.textfield.focusBorderColor.cgColor` needs to be set after `delegate?.textField(textField, shouldChangeCharactersIn: range, replacementString: string)` logic is finished

Reply:
My bad, thanks for explaining this 👍